src/detect-engine-threshold.c

```diff
@@ -69,6 +69,11 @@
 #include "util-var-name.h"
 #include "tm-threads.h"
 
+#define TIME_DIFF_SEC(p, lookup_tsh) \
```

Comment:
My preference is to either
1. Pass `p->ts` instead of `p`
2. Or, call this `PKTTIME_DIFF_SEC` to denote it's focused utility.

Reply:
I went with Option 1 in the v3 PR. Thanks!